Modules/Filtering/Thresholding/include/itkHistogramThresholdImageFilter.h

```diff
@@ -59,22 +50,23 @@ namespace itk
  */
 
 template <typename TInputImage, typename TOutputImage, typename TMaskImage = TOutputImage>
-class ITK_TEMPLATE_EXPORT HistogramThresholdImageFilter : public ImageToImageFilter<TInputImage, TOutputImage>
+class ITK_TEMPLATE_EXPORT HistogramThresholdImageFilter
+  : public HistogramThresholdImageFilterBase<TInputImage, TOutputImage>
```

Comment:
Shouldn't the `TMaskImage` template parameter be passes to the parent class.

Reply:
My bad. Thanks Brad. Done in 624c503.